src/screens/Dashboard/WalletPanel/index.js

```diff
@@ -40,8 +40,8 @@ const WalletPanel = ({ style, wallets } : Props) => ((
             <Text style={styles.body}>{i18n.t('screens.dashboard.walletPanel.empty')}</Text>
           :
             <View>
-              <Text style={styles.footnote}>ETH</Text>
-              <Text style={styles.currencyLarge}>{prettyWalletBalance(wallets[0])}</Text>
+              <Text style={styles.footnote}>XPAT</Text>
+              <Text style={styles.currencyLarge}>{prettyWalletBalance(wallets[1])}</Text>
```

Comment:
I'm not sure if it's a good idea to use wallet by index, since it can be that PAT wallet is not one with index 1. I guess you'd better search a wallet by currency from all wallets to get PAT wallet.

Reply:
So far this is by index because we are creating the wallets manually and know for sure their index (this is related to the way the ethereum service works and the changes needed for dropping the Panthalassa libs here). At the moment we add new wallets, maybe we could search by currency better